src/views/messages/container.jsx

```diff
@@ -26,14 +25,17 @@ var Messages = React.createClass({
         };
     },
     componentDidUpdate: function (prevProps) {
-        if (this.props.user != prevProps.user) {
+        if (this.props.user.username !== prevProps.user.username) {
```

Comment:
Just wondering why you needed to add `.username`

Reply:
seemed quicker to check username (string) equality than user (object) equality, if that makes sense